docs/gke-installation.md

```diff
@@ -0,0 +1,126 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in noEncap mode, in which no encapsulation is required for any kind of traffic (Intra Node,
```

Comment:
To be consistent with Su's implementation, call it NetworkPolicy only mode.

Reply:
Actually I want to highlight noEncap mode like Calico does in its documentation too. Can we keep this as is.  Also, would @antoninbas be updating getting started guide, or I should do it?

Reply:
you should upload all the relevant documentation as part of this PR

Reply:
noEncap mode is confusing as in our implementation it means different thing. I think we should use policy only mode, but you can explain about no encapsulation.

Reply:
Done. Changed to NetworkPolicy only mode.